projects/block-storage-service-typescript/src/block-storage.ts

```diff
@@ -0,0 +1,141 @@
+import * as path from "path";
+import * as mkdirp from "mkdirp";
+import * as levelup from "levelup";
+import leveldown from "leveldown";
+
+import { logger, config, types, topology, topologyPeers, CryptoUtils, Encoding } from "orbs-common-library";
+
+const crypto = CryptoUtils.loadFromConfiguration();
+
+export default class BlockStorage {
+  // The path to the local LevelDB will be of the format: "../../db/[ENVIRONMENT]/blocks_[NODE].db".
+  public static readonly LEVELDB_NAME: string = `blocks_${crypto.whoAmI()}.db`;
```

Comment:
Why do we need different file names per node?

Reply:
We don't. Good point.